resources/nats-streaming/templates/uninstall-natss-namespace.yaml

```diff
@@ -0,0 +1,112 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# About:
+# This post-delete job removes the knative-eventing namespace
+# It does so by using a job which is executed after the helm chart is deleted
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-namespace-delete
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    # Executes on a deletion request before any resources are deleted from Kubernetes
+    # this gives the controllers a chance to act on CRs with a finalizer
+    # source: https://helm.sh/docs/topics/charts_hooks/
+    helm.sh/hook: post-delete
+    helm.sh/hook-delete-policy: "hook-succeeded,before-hook-creation"
+    # everything without a dependency has weight 0
+    helm.sh/hook-weight: "0"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-namespace-delete
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    helm.sh/hook: post-delete
+    helm.sh/hook-delete-policy: "hook-succeeded,before-hook-creation"
+    helm.sh/hook-weight: "0"
+rules:
+  - apiGroups: [""]
+    resources: ["namespaces"]
+    verbs: ["list", "delete"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Release.Name }}-namespace-delete
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    helm.sh/hook: post-delete
+    helm.sh/hook-delete-policy: "hook-succeeded,before-hook-creation"
+    helm.sh/hook-weight: "0"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Release.Name }}-namespace-delete
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Release.Name }}-namespace-delete
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-namespace-delete
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    helm.sh/hook: post-delete
+    helm.sh/hook-delete-policy: "hook-succeeded,before-hook-creation"
+    sidecar.istio.io/inject: "false"
+    # dependent on clusterrole, clusterrolebinding, serviceaccount
+    # => hook-weight=min(hook-weight)+1
+    helm.sh/hook-weight: "1"
+spec:
+  backoffLimit: 1
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      name: {{ .Release.Name }}-namespace-delete
+    spec:
+      serviceAccountName: {{ .Release.Name }}-namespace-delete
+      restartPolicy: Never
+      containers:
+        - name: {{ .Release.Name }}-namespace-delete
+          image: eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20200507-070ff576
+          command:
+            - "/bin/sh"
+          args:
+            - "-c"
+            - |
+              set -eux -o pipefail
+              echo "starting deletion of {{ .Release.Namespace }} namespace"
+              kubectl \
+                delete \
+                namespace \
+                {{ .Release.Namespace }}
```

Comment:
why
not
on
a
single
line
?

Reply:
Why not on multiple lines? :-P Every command is on multiple lines here, so at least it is consistent in the PR :-D But honestly, I don't mind about having it on one line. It would be nice to have some style guidelines here.